HISTORY.rst

```diff
@@ -39,6 +39,10 @@ Unreleased Changes
 * General
     * Update python packaging settings to exclude a few config files and the
       workbench from source distributions and wheels
+    * Updating SDR test values due to an update in GDAL's mode resampling
```

Comment:
I'm assuming this is in here because we haven't merged the latest `main` into `natcap:feature/urban-nature-access`

Reply:
Yep!  That's all this is.